features/table.yml

```diff
@@ -66,3 +66,15 @@ compat_features:
   - html.elements.th.scope
   - html.elements.thead
   - html.elements.tr
+  - css.properties.border-collapse
+  - css.properties.border-collapse.collapse
+  - css.properties.border-collapse.separate
+  - css.properties.border-spacing
+  - css.properties.caption-side
+  - css.properties.caption-side.bottom
+  - css.properties.caption-side.bottom-outside
+  - css.properties.caption-side.top
+  - css.properties.caption-side.top-outside
+  - css.properties.caption-side.writing-mode_relative_values
+  - css.properties.empty-cells
+  - css.properties.table-layout
```

Comment:
These CSS properties apply specifically to table elements or sub-elements. They could conceivably be split into a table styles feature or two features, related to border styles and caption styles. This seemed the most straightforward